earth2mip/inference_medium_range.py

```diff
@@ -60,13 +55,15 @@ def _mean(self, x):
 
     def call(self, truth, pred):
         xy = self._mean((truth - pred) ** 2)
-        return xy.cpu()
+        return (xy.cpu(),)
 
     def gather(self, seq):
         return torch.sqrt(sum(seq) / len(seq))
 
 
 class ACC:
```

Comment:
Similarly this doesn't actually compute ACC rather the essential statistics... maybe these need to have a call then some finalize call that computes the results at the end?

Reply:
That was what .gather did, but I moved that time_average_metrics.

I think these classes overall are a bit over-engineered. I had intended to add more metrics but never did.